plugin/advisor/mysql/mysql_migration_compatibility_advisor.go

```diff
@@ -60,59 +60,59 @@ type compatibilityChecker struct {
 
 func (v *compatibilityChecker) Enter(in ast.Node) (ast.Node, bool) {
 	code := common.Ok
+	switch node := in.(type) {
 	// DROP DATABASE
-	if _, ok := in.(*ast.DropDatabaseStmt); ok {
+	case *ast.DropDatabaseStmt:
 		code = common.CompatibilityDropDatabase
-		goto END
-	}
 	// RENAME TABLE
-	if _, ok := in.(*ast.RenameTableStmt); ok {
+	case *ast.RenameTableStmt:
 		code = common.CompatibilityRenameTable
-		goto END
-	}
 	// DROP TABLE/VIEW
-	if _, ok := in.(*ast.DropTableStmt); ok {
+	case *ast.DropTableStmt:
 		code = common.CompatibilityDropTable
-		goto END
-	}
+	// CREATE UNIQUE INDEX
+	case *ast.CreateIndexStmt:
```

Comment:
nit: The order is intended since we would like to group logically similar stmt together, thus we put indexstmt after tablestmt

Reply:
Fixed.